src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -181,7 +185,17 @@ def _fill_ftp_publishing_url(webapp, resource_group_name, name, slot=None):
 
 def _add_linux_fx_version(resource_group_name, name, custom_image_name):
     fx_version = '{}|{}'.format('DOCKER', custom_image_name)
-    update_site_configs(resource_group_name, name, linux_fx_version=fx_version)
+    return update_site_configs(resource_group_name, name, linux_fx_version=fx_version)
+
+
+def _delete_linux_fx_version(resource_group_name, name):
+    fx_version = ''
+    return update_site_configs(resource_group_name, name, linux_fx_version=fx_version)
+
+
+def _get_linux_fx_version(resource_group_name, name):
+    site_config = get_site_configs(resource_group_name, name)
+    return str(site_config.linux_fx_version)
```

Comment:
What is the type of `site_config.linux_fx_version` which requires a conversion to `str`. I assume it is a string type already, no?